tests/test_query.py

```diff
@@ -41,19 +46,45 @@ def test_query_should_add_language(self):
     def test_query_should_add_start_time(self):
         q = self._client.query()
         q.pattern = "spotify"
-        q.start_time = datetime.datetime(2012, 12, 28, 9, 1, 22)
+        q.start_time = self.datetime_with_timezone(datetime(2012, 12, 28, 9, 1, 22), "UTC")
         self.assertEqual(q.request_parameters()['ts'], "2012-12-28 09:01:22")
 
+    def test_query_using_start_time_without_timezone(self):
+        q = self._client.query()
+        q.pattern = "spotify"
+        q.start_time = datetime(2012, 12, 28, 9, 1, 22)
+        with self.assertRaises(twingly_search.TwinglyQueryException):
```

Comment:
Ah, thought I had seen it somewhere. Anyway, what does our other libraries do? I was under the impression we don't raise, but I haven't closely followed the latest changes and I can't test now.

To me, at least, it makes sense to assume no tzinfo => UTC.